### PR TITLE
Revamped Contact Page + Moved Where to Find Us to About Tab from Contact Page

### PR DESCRIPTION
**Pull request by evanhearne**
# Description

Titles for contact info has been replaced with hyperlinked icons, a Where to Find Us page has been made and added to the About Tab.

## What

-Hyperlinked icons replaced contact info titles
-Where to Find Us page has been made, added to About tab
-Navigation bar has been updated to reflect changes
-Tab image has been added to contact.html and calendar.html

## Why

Contact page looked a bit messy, and a bit bland. The page now has hyperlinked icons to reflect modern users of internet - phones. Where to Find Us didn't really blend with the contact page, so it now has its own dedicated page in the About Tab.

## Verification Steps

http-server, go to About --> Where to Find Us and ensure that a page comes up with a map preview. http-server, go to Contact and ensure that a page comes up with hyperlinked icons, besides school code.

# Checklist:

- [x] This pr is linked to the related trello card
- [x] My code follows the style guidelines of this project
- [x] I have requested a code review from (at least) two team members
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I have made corresponding changes to the documentation
- [ ] This change has been verified by another team member

### CODE REVIEW

**Comment by evanhearne** [comment]
@laurafitzgerald @denispop9 could you please take a look at this PR when you have a chance? Thanks :)

File: pages/contact.html [hunk 1]
```diff
@@ -26,20 +27,15 @@
...
                     <div class="ui center aligned basic segment">
-                        <p>Email School:<a href="mailto:info@mountsioncbssecondary.ie" target="_top">info@mountsioncbssecondary.ie</a></p>
-                        <p>Web: mountsioncbssecondary.ie</p>
-                        <p>Telephone Number: 051 377378</p>
-                        <p>Fax Number: 051-376468</p>
-                        <p>School Roll No. 64930i</p>
-                        <p>Address:</p>
-                        <p>Mount Sion CBS Secondary School,</p>
+                        <p><a href="mailto:info@mountsioncbssecondary.ie"><i class="envelope link icon"></a></i>info@mountsioncbssecondary.ie</p> 
+                        <p><a target="_blank" href="http://www.mountsioncbssecondary.ie"><i class="globe icon"></i></a>mountsioncbssecondary.ie</p>
```
Comment (laurafitzgerald) [inline pages/contact.html]: @evanhearne what do you think about removing this one? Once we publish this site, we'll be removing the old one so we probably don't want to link out to it. wdyt?

File: pages/contact.html [hunk 2]
```diff
@@ -26,20 +27,15 @@
...
+                        <p><a href="tel:+35351377378"><i class="phone icon"></i></a>051 377378</p>
+                        <p><a href="tel:+35351376468"><i class="fax icon"></a></i>051 376468</p>
```
Comment (laurafitzgerald) [inline pages/contact.html]: For the fax we probably don't need to provide a telephone link as people can't ring the fax number. The icon looks great though.

File: pages/contact.html [hunk 3]
```diff
@@ -26,20 +27,15 @@
...
+                        <p><i class="building icon"></i>School Roll Number: 64930I</p>
+                        <p><a target="_blank" href="https://www.google.com/maps?ll=52.256315,-7.116627&z=14&t=m&hl=en-US&gl=IE&mapclient=embed&cid=17885305901261910436"><i class="address card icon"></a></i>Mount Sion CBS Secondary School,</p>
```
Comment (laurafitzgerald) [inline pages/contact.html]: When I viewed this I didn't immediately know that the address below it was part of it. Perhaps you could add a line above it to show that the three lines here are part of the address.

File: pages/about/Calendar.html [hunk 4]
```diff
@@ -5,6 +5,7 @@
     <link href="./assets/favicon.ico" rel="icon" type="../../image/x-icon">
     <link href="../../semantic/out/semantic.min.css" rel="stylesheet" type="text/css">
     <link href="../../assets/css/style.css" rel="stylesheet" type="text/css">
+    <link href="../../assets/favicon.ico" rel="icon" type="assets/image/x-icon">
```
Comment (laurafitzgerald) [inline pages/about/Calendar.html]: Maybe I missed it but where does this icon get used?

File: pages/contact.html (hunk 1 above)
Comment (evanhearne) [reply pages/contact.html]: Yes, @laurafitzgerald that sounds good. I will remove that now.

File: pages/contact.html (hunk 2 above)
Comment (evanhearne) [reply pages/contact.html]: OK, so you are saying just to have the icon, not the hyperlink, right?

File: pages/contact.html (hunk 3 above)
Comment (evanhearne) [reply pages/contact.html]: Yes, I will do that now.

File: pages/about/Calendar.html (hunk 4 above)
Comment (evanhearne) [reply pages/about/Calendar.html]: @laurafitzgerald it is used in the tab image on the web browser.

**Comment by evanhearne** [comment]
@laurafitzgerald changes have been made, please review when you have the time. :)

File: pages/contact.html (hunk 2 above)
Comment (laurafitzgerald) [reply pages/contact.html]: Yes exactly.

File: pages/about/Calendar.html (hunk 4 above)
Comment (laurafitzgerald) [reply pages/about/Calendar.html]: Ah was it just missing from this page?

**Comment by laurafitzgerald** [comment]
@evanhearne reviewing the changes now.

File: pages/contact.html (hunk 2 above)
Comment (laurafitzgerald) [reply pages/contact.html]: @evanhearne after the review the fax number is still a link

**Comment by evanhearne** [comment]
@laurafitzgerald could I get an update on this?

File: pages/about/Calendar.html (hunk 4 above)
Comment (evanhearne) [reply pages/about/Calendar.html]: @laurafitzgerald yes.

File: pages/contact.html (hunk 2 above)
Comment (evanhearne) [reply pages/contact.html]: @laurafitzgerald amended this.

**Pull request merged**